backend/BackendMIIT/src/main/java/com/example/BackendMIIT/service/impl/IndividualAchievementsServiceImpl.java

```diff
@@ -0,0 +1,140 @@
+package com.example.BackendMIIT.service.impl;
+
+import com.example.BackendMIIT.configuration.UrlsConfig;
+import com.example.BackendMIIT.model.domain.IndividualAchievements;
+import com.example.BackendMIIT.repositories.IndividualAchievementsRepository;
+import com.example.BackendMIIT.service.IndividualAchievementsService;
+import lombok.SneakyThrows;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@Service
+public class IndividualAchievementsServiceImpl implements IndividualAchievementsService {
+
+    private final IndividualAchievementsRepository individualAchievementsRepository;
+    private final UrlsConfig urlsConfig;
+
+    public IndividualAchievementsServiceImpl(IndividualAchievementsRepository individualAchievementsRepository, UrlsConfig urlsConfig) {
+        this.individualAchievementsRepository = individualAchievementsRepository;
+        this.urlsConfig = urlsConfig;
+    }
+
+    @Override
+    public void parseAndSaveAchievements() {
+        Set<IndividualAchievements> parsedAchievements = parse();
+
+        parsedAchievements.add(new IndividualAchievements(
+                "Статус чемпиона или призера Олимпийских, Паралимпийских, " +
+                        "Сурдлимпийских игр, чемпионатов мира и Европы.", "10"));
+        parsedAchievements.add(new IndividualAchievements(
+                "Наличие документа об образовании с отличием " +
+                        "(аттестат или диплом).", "10"));
+        parsedAchievements.add(new IndividualAchievements(
+                "Наличие золотого, серебряного или бронзового знака отличия ГТО." +
+                        "Начисление баллов за наличие знака ГТО осуществляется однократно.", "2"));
+        parsedAchievements.add(new IndividualAchievements(
+                "Участие в добровольческих формированиях в рамках " +
+                        "специальной военной операции.", "10"));
+
+        individualAchievementsRepository.saveAll(parsedAchievements);
+    }
+
+    @Override
+    public List<IndividualAchievements> getAllIndividualAchievements() {
+        return individualAchievementsRepository.findAll();
+    }
+
+    @SneakyThrows
+    public Set<IndividualAchievements> parse() {
+        String iaUrl = urlsConfig.getIa();
+
+        Set<IndividualAchievements> achievements = new HashSet<>();
+
+        try {

```

Comment:
You don't have to use try-catch, because you're using SneakyThrows

Reply:
Oh, really. I forgot remove try-catch :D

Reply:
Done